lib/matplotlib/axes/_axes.py

```diff
@@ -5748,9 +5748,9 @@ def _pcolorargs(self, funcname, *args, shading='auto', **kwargs):
 
         if shading == 'flat':
             if (Nx, Ny) != (ncols + 1, nrows + 1):
-                raise TypeError('Dimensions of C %s are incompatible with'
-                                ' X (%d) and/or Y (%d); see help(%s)' % (
-                                    C.shape, Nx, Ny, funcname))
+                raise TypeError(f"Dimensions of X({Nx}) and y({Ny}) should"
```

Comment:
Y should be capitalized.  Suggest "Relevant" because X and Y can be 2-D....  Otherwise this looks good to me...
```suggestion
                raise TypeError(f"Relevant dimensions of X({Nx}) and Y({Ny}) should"
```

Reply:
Ok Sir, Thank you for the suggestion